app/models/user.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+class User extends AppModel
+{
+    public function getUserDetail($id)
+    {
+        $db = DB::conn();
+        $row = $db->row('SELECT * FROM user WHERE id = ?', array($id));
+        return $row;
```

Comment:
since this is in the class, the return should be instance of itself po, and access it by `->` such as `$user->name`, `$user->email` etc. you can use a converter function in helper po to return the the object as array